docs/guides/blocks/formulas.md

```diff
@@ -30,20 +32,74 @@ Formulas can accept parameters, which is the primary way for them to access data
 
 ## Results
 
-TODO
+All formulas must return a result, which is a single value matching the type specified in the `resultType` property of the formula definition. See the [Data types guide][data-types] for more information on the type of values that can be returned.
+
+
+## Authentication
+
+Formulas can use [authentication][authentication] to fetch private data. When using [system authentication][system_auth] there is no change to the user experience, but when you use [user authentication][user_auth] the formula editor will be automatically updated to prompt for a connected account. The connected account will appear as the first parameter of the formula, but it's value is not passed to your formula's `execute` method.
+
+=== "Formula editor"
+    <img src="../../../images/formula_account.png" srcset="../../../images/formula_account_2x.png 2x" class="screenshot" alt="Account parameter in formula editor">
+
+=== "Code"
+    ```ts
+    pack.setUserAuthentication({
+      type: coda.AuthenticationType.HeaderBearerToken,
+      instructionsUrl: "https://todoist.com/app/settings/integrations",
+    });
+
+    pack.addFormula({
+      name: "GetTask",
+      description: "Gets a Todoist task by URL",
+      parameters: [
+        coda.makeParameter({
+          type: coda.ParameterType.String,
+          name: "url",
+          description: "The URL of the task",
+        }),
+      ],
+      // ...
+      execute: async function ([url], context) {
+        // ...
+      },
+    });
+    ```
+
+By default if a Pack defines an authentication method then all formulas will require a connected account. You can change this behavior using the [`connectionRequirement`][connectionRequirement] property of the formula definition. Setting it to `Optional` makes the user account optional, and `None` removes the account parameter completely.
+
+While it's possible to use multiple accounts within a document, each instance of the formula must use a specific account. It isn't possible to have a formula that uses the account of whichever user is currently viewing the document. It's important for collaboration that all users in a Coda doc have access to the same data.
```

Comment:
This last part isn't actually true, it depends on whether the formula is an action or not. If a formula has `isAction: true` you can call it with "User's private account".

Reply:
Ya, my thought with this guide was for it to describe the behavior of "regular" formulas (not actions or syncs), and then in the actions guide to describe the ways in which action formulas differ from regular formulas. I felt like it was easier to be definitive in this guide vs. having to qualify "except in this case" for the other formula types. I've added a note at the top of this guide to make that distinction clearer, but let me know if you think it would be better to include the caveats throughout.

Reply:
Ok let's see how it goes. Can add the caveats in later if users are missing the distinction.